src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+let employeesList = [...document.querySelectorAll('li')];
+let forget = employeesList;
+const helper = (list) => {
+  return list.map((x) => {
+    return {
+      salary: Number(x.dataset.salary.split(',').join('').slice(1)),
+      element: x,
+    };
+  });
+};
+
+employeesList = helper(employeesList);
+
+const sortList = (list) => {
+  return list.sort((a, b) => b.salary - a.salary);
+};
+
+employeesList = sortList(employeesList);
+
+const parentOfList = document.querySelector('ul');
+
+parent.innerHTML = '';
```

Comment:
There is a reference to `parent`, which is not defined. It should be `parentOfList` to match the variable declared on line 22.